public/api/documentation/1.0/State-Pension.xml

```diff
@@ -352,6 +352,58 @@
                     <cell><code>PS</code></cell>
                     <cell>A user with the PostStatePensionAge exclusion</cell>
                 </row>
+                <row>
+                    <cell><code>AM</code></cell>
+                    <cell>A user with the AmountDissonance exclusion</cell>
+                </row>
+                <row>
+                    <cell><code>AB</code></cell>
+                    <cell>A user with the Abroad exclusion</cell>
+                </row>
+                <row>
+                    <cell><code>MA</code></cell>
+                    <cell>A user with the IsleOfMan exclusion</cell>
+                </row>
+                <row>
+                    <cell><code>MW</code></cell>
+                    <cell>A user with the MarriedWomenReducedRateElection exclusion</cell>
+                </row>
+                <row>
+                    <cell><code>MX</code></cell>
+                    <cell>A user that is excluded for multiple reasons</cell>
+                </row>
+                <row>
+                    <cell><code>CR</code></cell>
+                    <cell>A user with COPE</cell>
+                </row>
+                <row>
+                    <cell><code>PP</code></cell>
+                    <cell>A user with a protected payment</cell>
+                </row>
+                <row>
+                    <cell><code>CA</code></cell>
+                    <cell>A user below the minimum qualifying period that cannot get any State Pension even by working to pension age and filling payable gaps</cell>
+                </row>
+                <row>
+                    <cell><code>AL</code></cell>
+                    <cell>A user below the minimum qualifying period that can get some State Pension by working to pension age and also filling payable gaps</cell>
```

Comment:
A user below the minimum qualifying period who can get some State Pension by working until State Pension age and also filling payable gaps